litestar/middleware/session/base.py

```diff
@@ -241,5 +252,6 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
 
         connection = ASGIConnection[Any, Any, Any, Any](scope, receive=receive, send=send)
         scope["session"] = await self.backend.load_from_connection(connection)
+        connection.set_session_id(self.backend.get_session_id(connection))
```

Comment:
Related to what I mentioned above. The setting of the session ID should be done manually.

Reply:
Okay. I added this method to provide a public interface for setting property in `connection._connection_state`. I expect mypy or pyright to complain when I replace this with `connection._connection_state.session_id`, so I assume it's okay to add a linting exception in that case, right?